osrsbox/items_api/item_equipment.py

```diff
@@ -21,60 +21,46 @@
 
 import collections
 from typing import Dict
-from typing import List
 
 
 class ItemEquipment:

```

Comment:
To me, this class is starting to look a lot like a dataclass. If you want, we can turn it into a dataclass. Would only need to add the backport library to the requirements.txt and the setup.py to run: https://pypi.org/project/dataclasses/

Benefits to this are that we get all the autogenerated methods for free and it requires typing on the class, so we will have static typing to validate. Cons are the time for conversion and adding an additional dependency when installing the package on Python 3.6, since Python 3.7 supports dataclasses natively.

I would convert this to a dataclass but it is definitely not necessary.

Some resources if you're unfamiliar:
https://docs.python.org/3/library/dataclasses.html
https://www.youtube.com/watch?v=T-TwcmT6Rcw